src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+
+  if (days >= 7) {
+    return (dayPrice * days) - 50;
```

Comment:
you have multiplied dayPrice and days 3 times. use the variable for this action.